src/test/lib/DataTest.cpp

```diff
@@ -3376,6 +3376,9 @@ QuicTestStreamReliableReset(
     TEST_TRUE(Listener.LastConnection->HandshakeComplete);
     CxPlatSleep(50); // Wait for things to idle out
 
+    //
+    // Note: we shouldn't be testing dropped packets during Release builds.
+    //
```

Comment:
Let's go ahead and fix this part up. The easiest way is to just do the for-loop and `BitmapLossHelper` in `DEBUG` mode only. Otherwise, just do one loop without loss in Release mode.

Reply:
Done! 91f4021db5f2d340a613e505675611cb83c29e18